packages/backend/src/modules/liveness/LivenessModule.ts

```diff
@@ -54,6 +54,8 @@ export function createLivenessModule(
   const livenessController = new LivenessController(
     livenessRepository,
     config.projects,
+    clock,
+    livenessConfigurationRepository,
```

Comment:
pass logger and send error to sentry when sanity check fails (.errror() method call)

Reply:
[eb0acbf](https://github.com/l2beat/l2beat/pull/2267/commits/eb0acbf5cf87858018c365659a6d441d7db67f3c)